tutorials/btp-integration-suite-execute-batch-payload/btp-integration-suite-execute-batch-payload.md

```diff
@@ -0,0 +1,377 @@
+
+Then, **Send** the following query from Postman:
+
+`select * from samplejdbctest`
+
+  <!-- border -->![Atomic Batch Operation Verification](Verify-Atomic.png)
+
+This response shows that no new data has been inserted. Hence, verifying the Atomic operation.
+
+### Non-Atomic Batch Operation
+
+
+Depending upon your driver (which depends upon choice of database), the behaviour of Non- Atomic mode differs. Say for SQL server database, you would be able to insert the unique record and it would fail all the duplicate records. So, the complete payload doesn't fail. But for some other database and driver, it may act similar to Atomic mode and fail the payload even if single error is encountered.
+
+1. In your integration flow, edit the settings for JDBC Receiver adapter. Under **Connections** tab, select the **Batch Payload** checkbox and select **Batch Operation** as Non-Atomic.
+
+    <!-- border -->![Batch Payload Setting Non-Atomic](Batch-Payload-Setting-Non-Atomic.png)
+
+2. **Send** this payload from Postman. Here we are making the **`emp_id`** (of second record) as 4. Thus, making that record unique, but the first record is duplicate.
+
+    ```
+    <root>
+    	<insert_statement1>
+    		<dbTableName action="INSERT">
+    			<table> samplejdbctest</table>
+    			<access>
+    				<emp_id>1</emp_id>
+    				<emp_name hasQuot="Yes">v XXXXX</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement1>
+    	<insert_statement2>
+    		<dbTableName action="INSERT">
+    			<table>samplejdbctest</table>
+    			<access>
+    				<emp_id>4</emp_id>
+    				<emp_name hasQuot="Yes"> g28</emp_name>
+
+    			</access>
+    		</dbTableName>
+    	</insert_statement2>
+    </root>
+    ```
+      <!-- border -->![Non-Atomic Payload](Payload-Non-Atomic.png)
+
+      Check the MPL too.
+
+    <!-- border -->![Payload MPL](Payload-Processed-Non-Atomic.png)
+
+3. Let's confirm, whether the unique record (with `emp_id` 4) is inserted in the database or not. To do this, uncheck the **Batch Mode** checkbox and **Deploy** your integration flow.
+
+4. **Send** the following query from Postman:
+    `select * from samplejdbctest`  
+
+      <!-- border -->![Non-Atomic Batch Mode Verification](Verify-Non-Atomic.png)
+
+So, it is evident that using Non-Atomic mode, the new record is getting inserted and the duplicate record fails. So, the whole payload doesn't fail unlike Atomic Batch operation.
+
+>The function/behaviour of Non-Atomic Batch Operation may depend upon the choice of database.
+
+### Send Payload using Groovy Script
+
+You can also upload the payload using the **Groovy Script**. In this case you must use the prepared statements.
+
+
+1. Uncheck **Batch Mode** in the integration flow.
+<!-- border -->![Batch Mode Off](Batch-Off.png)
+Let's insert a new database table. We have already seen creating sample database via **Content Modifier**. Now we will do it directly from Postman. **Send** this query from Postman.
```

Comment:
Don't say "we" when addressing the author. Say "you" instead!

Reply:
done